### PR TITLE
Increase max length of voice messages to 15m (PSG-661)

### PR DESCRIPTION
**Pull request by Johennes**
Closes: https://github.com/vector-im/element-web/issues/18620
Relates to: vector-im/element-ios#5415

Notes: Increase max length of voice messages to 15m

Signed-off-by: Johannes Marbach <johannesm@element.io>

## Checklist

* [x] Tests written for new code (and old code if feasible)
* [x] Linter and other CI checks pass
* [x] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

---
Here's what your changelog entry will look like:

## ✨ Features
 * Increase max length of voice messages to 15m ([\#9133](https://github.com/matrix-org/matrix-react-sdk/pull/9133)). Fixes vector-im/element-web#18620.

### CODE REVIEW

**Comment by robintown** [comment]
Does this have product approval? Also, this would close https://github.com/vector-im/element-web/issues/18620

**Comment by Johennes** [comment]
> Does this have product approval? Also, this would close [vector-im/element-web#18620](https://github.com/vector-im/element-web/issues/18620)

Signed off by @jakewb-b in PSG-661, yes.

**Review by robintown — APPROVED** [review]
Code looks fine

**Comment by Marv93** [comment]
The comment in line 232, 233 refers to the 2 minute limit and could be changed to reflect the new time limit.

Also this PR relates to: https://github.com/vector-im/element-android/issues/4160

**Comment by t3chguy** [comment]
Could we please fix the changelog to not include internal project identifiers which don't mean anything to 99.99% of people

**Comment by Johennes** [comment]
> The comment in line 232, 233 refers to the 2 minute limit and could be changed to reflect the new time limit.

@Marv93 thanks. I tweaked that comment, too, now.

> Could we please fix the changelog to not include internal project identifiers which don't mean anything to 99.99% of people

@t3chguy sorry this PR was opened before our conversation on https://github.com/matrix-org/matrix-react-sdk/pull/9135. Truth be told, I used the Jira issue key in changelog entries across EW, EI and EA for months without receiving complaints so I didn't realize this was problematic. I've added a `Notes: ` clause to the description of this PR, too, now to remove the key. Sorry for the hassle.

**Comment by t3chguy** [comment]
https://riots.im/ shows only 2 instances of PSG and 2 instances of PSF - so either there's very few PS changes making it into Web or most of them lack this tag

File: src/audio/VoiceRecording.ts
Comment (robintown) [inline src/audio/VoiceRecording.ts:233]:
```suggestion
        // safety can allow us to overshoot the target a bit, but at least when we say 15min
```

Comment (Johennes) [reply src/audio/VoiceRecording.ts]: Darn! Thanks @robintown! 🙏

**Comment by Johennes** [comment]
Is this ok to merge now (I don't have write access)?

**Review by SimonBrandner — APPROVED** [review]

**Pull request merged**
